spec/board_spec.rb

```diff
@@ -116,4 +109,56 @@
       expect(cell_3.ship == cell_2.ship).to eq(true)
     end
   end
+
+  describe '#render' do
+    it 'prints a visual of the board' do
+      board = Board.new
+      cruiser = Ship.new("Cruiser", 3)
+      board.place(cruiser, ["A1", "A2", "A3"])
+
+      expect(board.render).to eq(
+        "  1 2 3 4 \nA . . . . \nB . . . . \nC . . . . \nD . . . . \n"
+      )
+      expect(board.render(true)).to eq(
+        "  1 2 3 4 \nA S S S . \nB . . . . \nC . . . . \nD . . . . \n"
+      )
+    end
+
+    it 'can render visual of board with hits/misses' do
+      board = Board.new
+      cruiser = Ship.new("Cruiser", 3)
+      board.place(cruiser, ["A1", "A2", "A3"])
+      board.cells["A1"].fire_upon
+
+      expect(board.render).to eq(
+        "  1 2 3 4 \nA H . . . \nB . . . . \nC . . . . \nD . . . . \n"
+      )
+      board.cells["A2"].fire_upon
+      board.cells["A3"].fire_upon
+      board.cells["D1"].fire_upon
+      expect(board.render).to eq(
+        "  1 2 3 4 \nA X X X . \nB . . . . \nC . . . . \nD M . . . \n"
+      )
+    end
+    it 'can render visual of board with multiple hits/misses/sink' do
+      board = Board.new
+      cruiser = Ship.new("Cruiser", 3)
+      submarine = Ship.new("Submarine", 2)
+      board.place(cruiser, ["A1", "A2", "A3"])
+      board.place(submarine, ["C3", "D3"])
+      board.cells["A1"].fire_upon
+      board.cells["A2"].fire_upon
+      board.cells["A3"].fire_upon
+      board.cells["D1"].fire_upon
+      board.cells["D3"].fire_upon
+
+      require "pry"; binding.pry
```

Comment:
Should clean up some of these tests (ie our pry is still here!) on our next branch.